src/core/control/ToolHandler.cpp

```diff
@@ -358,7 +358,7 @@ void ToolHandler::saveSettings() {
     for (auto&& tool: tools) {
         SElement& st = s.child(tool->getName());
         if (tool->hasCapability(TOOL_CAP_COLOR)) {
-            st.setIntHex("color", int(tool->getColor()));
+            st.setIntHex("color", int(uint32_t(tool->getColor())));
```

Comment:
Thought about that, and previously it was there over templates, but I decided to remove it, because we really want to have such casts explicit.

It would be better, to implement an `setUIntHex` function, or even replace it by overloaded `setValue` functions.

Reply:
It looks awkward, because it actually is. And I don't want to hide it. I actually want, that someone will refactor SElement :')

Reply:
:+1:  Makes sense.